doc/installing.md

```diff
@@ -82,6 +82,10 @@ If you want the current user to be able to interact with the LXD daemon, add it
 getent group lxd | grep -qwF "$USER" || sudo usermod -aG lxd "$USER"
 ```
 
+```{note}
+If your user was not already a member of the `lxd` group, you need to log out and back in for the change to take effect.
+```
+
```

Comment:
Even though it's only one sentence, we should reuse it in the different locations. In this case, I think it's easiest to add it to `doc/substitutions.yaml` and then include it with `{{must_start_new_session}}` (or similar).

Reply:
Interesting mechanism I wasn't aware of, thanks.